Exercises/exercise_3_join_validator.md

```diff
@@ -1,27 +1,180 @@
 # Exercise 3
 Join and leave the Axelar Network as a validator node.
 
-Convert an existing Axelar Network node into a validator by staking tokens. A validator participates in block creation, transaction signing, and voting.
+Convert an existing Axelar Network node into a validator by attaching Bitcoin and Ethereum nodes and staking tokens. A validator participates in block creation, transaction signing, and voting.
 
 ## Level
-Intermediate
+Advanced
 
 ## Disclaimer
 Axelar Network is a work in progress. At no point in time should you transfer any real assets using Axelar. Only use testnet tokens that you're not afraid to lose. Axelar is not responsible for any assets lost, frozen, or unrecoverable in any state or condition. If you find a problem, please submit an issue to this repository following the template.
 
 ## Prerequisites
 - Complete all steps from `README.md`
-- RPC endpoints for your Bitcoin and Ethereum nodes
 
 ## Useful links
 - Extra commands to query Axelar Network state: https://github.com/axelarnetwork/axelarate-community/blob/main/EXTRA%20COMMANDS.md
 
 ## Joining the Axelar testnet
 Follow the instructions in `README.md` to make sure your node is up to date and you received some test coins to your validator account.
 
-## Set up Bitcoin and Ethereum RPC nodes
-As an Axelar Network validator, your Axelar node will vote on the status of Bitcoin and Ethereum transactions. To do so, it needs to be configured with your Bitcoin and Ethereum nodes using the RPC endpoint.
+## Set up Bitcoin and Ethereum nodes
+As an Axelar Network validator, your Axelar node will vote on the status of Bitcoin and Ethereum transactions. To do so, you must first set up your Bitcoin and Ethereum testnet nodes, and then provide the RPC endpoints to Axelar.
```

Comment:
can you add general requirements from the Bitcoin/Ethereum nodes? 
for instance, copy-paste the info the the validator would need to fill-in in the config file 
(e.g., RPC endpoint, username/password, etc.).

Reply:
specify that these can be configured on other external machines. no need to configure on the same physical node.

Reply:
I can add the default RPC endpoints to use for btc eth (if they follow the instructions below exactly). Username password only applies to bitcoin, and it is set by the user and will be different for everyone.

Reply:
Currently, the instructions only support nodes being configured on the same local machine.

The config setup is different if the bitcoin/ethereum and axelar nodes are on different machines. I would have to test this out myself before adding instructions. (bitcoin eth on my macbook, axelar on personal laptop)

Also, the IP address of the bitcoin ethereum node machine could change upon restart, adding an extra step for some users (depending on ISP and other settings)